photon-api/src/main/scala/com/linkedin/photon/ml/function/svm/SingleNodeSmoothedHingeLossFunction.scala

```diff
@@ -99,7 +99,9 @@ object SingleNodeSmoothedHingeLossFunction {
    * @param configuration The optimization problem configuration
    * @return A new SingleNodeSmoothedHingeLossFunction
    */
-  def apply(configuration: GLMOptimizationConfiguration): SingleNodeSmoothedHingeLossFunction = {
+  def apply(
+    configuration: GLMOptimizationConfiguration,
+    index: Int = -1): SingleNodeSmoothedHingeLossFunction = {
```

Comment:
Same.

Reply:
In addition, parameter name and type should change (see below):
```
interceptIndexOpt: Option[Int] = None
```